frontend/src/metabase-lib/Dimension.ts

```diff
@@ -1472,11 +1472,11 @@ export class AggregationDimension extends Dimension {
 
     const dimension = aggregation.dimension();
     const field = dimension && dimension.field();
-    const { semantic_type } = field || {};
+    const { base_type, semantic_type } = field || {};
     return new Field({
       name: aggregation.columnName(),
       display_name: aggregation.displayName(),
-      base_type: aggregation.baseType(),
+      base_type: base_type ?? aggregation.baseType(),
```

Comment:
Quoting [the PR description](https://github.com/metabase/metabase/pull/28741#issue-1603298559)
> The current `base_type` set from `Aggregation` is plainly wrong. The issue referenced for instance, has the aggregation field as **Min of Created Date**. And the issue was that we say the `base_type` is `type/Float` instead of using the underlying field's `base_type` of `type/DateTime`